### DISCUSSION TITLE
Erro ao se conectar com servidor do banco de dados

### DISCUSSION THREAD

**Discussion by matteus-henryk** [balta-io/forum#27] [Tira Dúvidas] [answered]
Curso: [Acesso à dados com .NET, C#, Dapper e SQL Server ](https://balta.io/cursos/acesso-dados-csharp-net-dapper-sql-server);
Aula: Dapper => primeira consulta;

.NET 6;
Dapper 2.0.90;
Microsoft Data SqlClient 5.0.0;

O seguinte erro acontece quando executo o ``dotnet run``: ``Unhandled exception. Microsoft.Data.SqlClient.SqlException (0x80131904): A connection was successfully established with the server, but then an error occurred during the login process. (provider: SSL Provider, error: 0 - A cadeia de certificação foi emitida por uma autoridade que não é de confiança.)``

Código escrito de acordo com a aula: 
![image](https://user-images.githubusercontent.com/63485326/192374892-539880ca-acd5-4267-8eb9-8dc38afe6c9c.png)

**Comment by andrebaltieri** [comment, marked as answer]
@matteus-henryk bom dia!
Só seguir os passos descritos na sessão **Erros Comuns** do nosso artigo de SQL Server no Docker:
https://balta.io/blog/sql-server-docker
